File: Linguagem C++/Estrutura Repetitiva/DentroFora.cpp
#include <iostream>

using namespace std;

int main()
{
    int n,x,contDentro,contFora;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    contDentro = 0;
    contFora = 0;
    for (int i=n;i<=x;i++){
        cout << "Digite um numero: ";
        cin >> x;
        if (x>=10 && x<=20){
           contDentro++;
        }else {
           contFora++;
        };
    };

    cout << "DENTRO = " << contDentro << endl;
    cout << "FORA = " << contFora << endl;

    return 0;
}

File: Linguagem Java/Exercicios/DentroFora.java
import java.util.Locale;
import java.util.Scanner;

public class DentroFora {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n,x,contDentro,contFora;
		
		contDentro = 0;
		contFora = 0;
		System.out.print("Quantos numeros voce vai digitar? ");
        n = sc.nextInt();
        
        for (int i=1;i<=n;i++) {
        	System.out.print("Digite um numero: ");
        	x = sc.nextInt();
        	
        	if (x >= 10 && x <= 20) {
        		contDentro++;        		
        	}else {
        		contFora++;        		
        	};
        };
        
        System.out.println(contDentro+" DENTRO");
        System.out.println(contFora+" FORA");
        
        sc.close();
	}

}
